CONTRIBUTING.md

```diff
@@ -24,20 +24,25 @@ serving the tests of Nmstate.
 
 - `./examples/` - Contains YAML examples for different configurations.
 
-- `./libnmstate/ifaces/` Contains the classes including the attributes of the 
+- `./logo` Logos used for publication.
+
+- `./k8s` k8s folder holds the scripts for kubernetes-nmstate CI
+
+- `./rust/src/clib` C binding of nmstate written in rust
+
+- `./rust/src/go/nmstate` Contains the classes including the attributes of the 
 interfaces.
  
-- `./libnmstate/nispor/` Contains the nmstate nispor plugin, currently for 
+- `./rust/src/python/libnmstate`, `./rust/src/python` Python library of nmstate written in python wrapping the C binding of nmstate
+ 
+- `./rust/src/lib/nispor/` Contains the nmstate nispor plugin, currently for 
```

Comment:
Plugins do not exist anymore. `Contains the code related to querying the network state with via Nispor`